models/repo/search.go

```diff
@@ -5,20 +5,72 @@ package repo
 
 import "code.gitea.io/gitea/models/db"
 
+// Strings for sorting result
+const (
+	// only used for repos
+	SearchOrderByAlphabetically        db.SearchOrderBy = "owner_name ASC, name ASC"
```

Comment:
Every `const` here is only used once for constructing the "map"

I think all of these `const` should be removed, and just fill the value into the map directly.

Reply:
well I"ll merge this now ... just create a pull and ping me in it :)